contracts/BaseWrapper.sol

```diff
@@ -179,7 +179,8 @@ abstract contract BaseWrapper {
 
                     // Limit amount to withdraw to the maximum made available to this contract
                     uint256 shares = Math.min(estimatedShares, availableShares);
-                    withdrawn = withdrawn.add(vaults[id].withdraw(shares));
+                    // If shares to be withdrawn is 0, do not try to withdraw (as it will revert)
+                    withdrawn = shares == 0 ? withdrawn : withdrawn.add(vaults[id].withdraw(shares));
```

Comment:
`availableShares` is > 0 (see above guard clause), and `amount > withdrawn` should also be true (see below `break` clause)

Reply:
I thought the same. However for some reason tests were not passing before adding that switch...

Reply:
Can you point to this issue in a CI failure?

Reply:
got it working without this fix.

resolving